sdk/storage/azure-storage-common/inc/azure/storage/common/paged_response.hpp

```diff
@@ -0,0 +1,50 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// SPDX-License-Identifier: MIT
+
+#pragma once
+
+#include <cstdlib>
+#include <string>
+
+#include <azure/core/context.hpp>
+#include <azure/core/http/raw_response.hpp>
+
+namespace Azure {
+
+template <class Derived> class PagedResponse {
+  bool m_hasMore = true;
+
+public:
+  std::string NextPageToken;
+  std::string CurrentPageToken;
+  std::unique_ptr<Azure::Core::Http::RawResponse> RawResponse;
+
+  bool HasMore() const { return m_hasMore; }
+
+  void NextPage(const Azure::Core::Context& context)
```

Comment:
Can you please file an issue for core to add API docs to this type once it moves down to Azure.Core.

Reply:
https://github.com/Azure/azure-sdk-for-cpp/issues/2118

Reply:
This field should have a default value.